sessions/nowcasting.qmd

```diff
@@ -2,18 +2,23 @@
 title: "Nowcasting concepts"
 order: 6
 ---
-
-[Introduction to nowcasting](slides/introduction-to-nowcasting)
-
 ```{r echo = FALSE}
 set.seed(123)
 ```
 
-# Objectives
+## Introduction
+
```

Comment:
This one has a different structure (in terms of introducing the slides) to the others